Simplified generic examples and consolidated imports for sdk examples in docs.

bevis-hp opened the pull request:
Working with @phazonoverload this makes the generic examples more generic and simple to avoid confusion with the working examples.

Also the imports for the SDK snippets have been consolidated into one root import.

A few examples have also been swapped around as they were in the wrong sections.

Commit:
Simplified generics and imports for items page snippets

Commit:
Simplified generics and imports for files page snippets

Commit:
Fixing simplified generic snippets in items page

Commit:
Simplified generics and imports for activity page snippets

Commit:
Simplified generics and imports for collections page snippets

Commit:
Simplified generics and imports for dashboards page snippets

Commit:
Simplified generics and imports for extensions page snippets

Commit:
Simplified generics and imports for fields page snippets

Commit:
Simplified generics and imports for flows page snippets

Commit:
Simplified generics and imports for folders page snippets

Commit:
Simplified generics and imports for notifications page snippets

Commit:
Simplified generics and imports for operations page snippets

Commit:
Simplified generics and imports for panels page snippets

Commit:
Simplified generics and imports for permissions page snippets

Commit:
Simplified generics and imports for presets page snippets

Commit:
Simplified generics and imports for relations page snippets

Commit:
Simplified generics and imports for relations page snippets

Commit:
Simplified generics and imports for revisions page snippets

Commit:
Simplified generics and imports for roles page snippets

Commit:
Consolidated imports for schema page snippets

Commit:
Simplified generics and imports for server page snippets

Commit:
Simplified generics and imports for settings page snippets

Commit:
Fixed mixed up snippets and simplified generics and imports for shares page snippets

Commit:
Simplified generics and imports for translation page snippets

Commit:
Fixed mixed up snippets and simplified generics and imports for user page snippets

Commit:
Simplified generics and imports fo uutilitie pages snippets

Commit:
Simplified generics and imports for webhook pages snippets

Commit:
Simplified generics and imports for authentication pages snippets

Commit:
Consolidated imports for query pages sdk snippets

phazonoverload commented:
Fab, thanks for this @be7DOTis. We'll take a look through it. 

Each endpoint has two togglers - one that's meant to give an overview of how it works, and the other giving a 'real world' example. In many cases, these were too similar and it was causing confusion. Just wanted to explain the motivator behind this body of work.

paescuj committed:
Merge branch 'main' into main

paescuj committed:
Format files

paescuj committed:
Update lockfile

paescuj committed:
Fix spelling

paescuj committed:
Format snippets

paescuj committed:
Aling `result` const

paescuj committed:
Small clean-ups

- Align `SEARCH` snippets, move "Learn more..." next to other hint
- ids -> IDs
- Other alignments

paescuj reviewed (approved):
LGTM!

![seal-of-approval2](https://github.com/directus/directus/assets/5363448/85e4063d-5865-4708-bd13-a5ecb4befe33)

phazonoverload reviewed (approved):
LGTM
